src/Symfony/Bundle/FrameworkBundle/Command/ContainerAwareCommand.php

```diff
@@ -53,4 +53,40 @@ public function setContainer(ContainerInterface $container = null)
     {
         $this->container = $container;
     }
+
+    /**
+     * Returns true if the service id is defined.
+     *
+     * @param string $id The service id
+     *
+     * @return bool true if the service id is defined, false otherwise
+     */
+    public function has($id)
+    {
+        return $this->container->has($id);
```

Comment:
$this->container might not be initialized. so it needs to be $this->getContainer